web_responsive/static/src/js/web_responsive.js

```diff
@@ -358,7 +367,24 @@ odoo.define('web_responsive', function (require) {
             } else {
                 this._super.apply(this, arguments);
             }
-        }
+        },
+    });
+
+    // Hide AppMenu & remove waiting anim when loaded action
+    ActionManager.include({
+        doAction: function () {
+            return this._super.apply(this, arguments).then(function () {
+                var $app_menu = $('.o_menu_apps .dropdown');
+                if ($app_menu.length) {
+                    if ($app_menu.hasClass('show')) {
+                        $app_menu.dropdown('toggle');
```

Comment:
I think you can just do `$app_menu.dropdown("hide")`, and avoid the `if` here.

Reply:
Only appears to be implemented the "toggle" method... :suspect: use "hide" or "show" prints: ```TypeError: No method named "hide"```

Reply:
Yikes, that's because Odoo 12 uses Bootstrap 4.1, and the `hide` method was implemented in 4.2.

OK, then could you add a TODO comment here indicating that, please?